src/Event.cc

```diff
@@ -31,7 +31,6 @@
  *
  */
 
-#include "boost/scoped_array.hpp"
```

Comment:
All you're doing here is removing an unused include -- you're not replacing it with anything. Please change the commit message to reflect that.